### ISSUE TITLE
[UPDATE ENTRY] Tesla API

### ISSUE THREAD

**Issue by Kristian-Buriasco** [unixorn/internet-of-trash#35]
## What list entry needs an update?
Tesla

## What should we update?
Remove it, or at least change it. It should be noted that Tesla is charging just for Fleet API access, it mostly impacts business and actually, it should make it cheaper for them to manage large fleets of Teslas. It will only impact you if you were using, as I understand it, the Tesla Fleet integration from HA. Reading the Integration's "Read Me" it also seems like before this change you were only allowed 200 requests per day, that now changes for the better because you are allowed for small applications to have a 10$ monthly discount, which calculates to about 324 daily requests. As I also said it should make it cheaper because before you would also be using the Fleet Telemetry API which costs more, now this is gathered through Tesla's "Case Studies" so I'm not so sure we can trust it but it does seem like a truthful one. Now please check if I understood the Tesla website correctly and don't take this as an Insult, I absolutely love the idea of having a list of bad companies but in this case it does not seem that tesla is one of them. Keep Up God's Work of listing the Worst companies out there!

If the vendor has resolved the issue, please add a link to a press release / blog post / article confirming it. If there's a new workaround, describe that here (with links to instructions if possible).
While if you were using the Tesla Fleet API there is the current workaround of changing your integration for Tesla data access, if you were using the Tesla integration ( which I think is from HACS ) there is nothing you should currently worry about.
Also if you were using the Tesla Fleed API as a business or anything else you should check the new pricing structure!

**Comment by lhovo** [comment]
https://github.com/alandtse/tesla reports
Tesla has deprecated the Owner API for most vehicles in favor of a new Fleet API
I am still using the Owner API, so I'm unsure how long this will be operating.

**Comment by Kristian-Buriasco** [comment]
Good point about the Owner API still being operational. It seems Tesla is allowing a grace period before fully transitioning to the Fleet API. For older cars and their users, the Owner API still works fine for now—let’s hope it stays that way for a while longer.
That said, there haven’t been any changes to Tesla Energy integrations, like Powerwall, which is worth pointing out. The phrasing in the readme—“Charging per API access”—could be misleading, as it might suggest that everything, including energy APIs, is now paid. In reality, most users likely won’t exceed the $10 monthly grace discount, so for small-scale applications, the change isn’t as drastic as it first sounds.
For those affected by the Fleet API shift, the Tesla HTTP Proxy offers a good workaround to maintain access to data without too much trouble. It’s also worth mentioning that this shift was likely a necessary step for newer cars, as their data seems to be accessible in a slightly different way now.
Hopefully, Tesla will clarify its plans soon. For now, though, it feels like we’re in a holding pattern with the Owner API. Fingers crossed!

**Comment by lhovo** [comment]
I agree on the Tesla Energy integrations support, I am longing for a way to control my power wall (charge/discharge) via home assistant without having to mess around with their schedule nonsense. Which is useless if the price of power changes every 5 minutes.

**Comment by unixorn** [comment]
I updated the wording in #40 - please take a look and let me know if it looks ok to y'all. If it needs further changes, please re-open this issue and add wording suggestions.

Thanks!
